src/main/kotlin/org/rust/lang/core/parser/RustParserUtil.kt

```diff
@@ -142,10 +142,13 @@ object RustParserUtil : GeneratedParserUtilBase() {
     @JvmField
     val ADJACENT_LINE_COMMENTS = WhitespacesAndCommentsBinder { tokens, _, getter ->
         var candidate = tokens.size
+        var containsDocComment = false
```

Comment:
Since you change parsing, don't forget to increase `org.rust.lang.core.parser.RustParserDefinition#PARSER_VERSION`